docs/vcf_to_bigquery.md

```diff
@@ -25,18 +25,25 @@ Our empirical results show that Avro files are more compressed compared to VCF f
 For example, [Illumina Platinum Genomes](https://www.illumina.com/platinumgenomes.html)
 are `30.66 GiB` in VCF format (located at
 [`gs://genomics-public-data/platinum-genomes`](https://console.cloud.google.com/storage/genomics-public-data/platinum-genomes/))
-while in Avro format they are `5.34 GiB`. In addition, as we see in the next section,
-Avro files can be loaded directly into BigQuery. These properties make Avro files a viable
-option for long-term storage of variant data.
+while in Avro format they are `5.34 GiB`. This compression does not apply to all VCF files,
+particularly joint genotyped VCF files bacause of multiple repeated samples are more
```

Comment:
s/bacause of multiple repeated samples.../, because multiple repeated samples...

Reply:
done.